classes/patterns/class-pageheader.php

```diff
@@ -0,0 +1,72 @@
+<?php
+/**
+ * Page Header pattern class.
+ *
+ * @package P4GBKS
+ * @since 0.1
+ */
+
+namespace P4GBKS\Patterns;
+
+/**
+ * Class Page Header.
+ *
+ * @package P4GBKS\Patterns
+ */
+class PageHeader extends Block_Pattern {
+
+	/**
+	 * Returns the pattern name.
+	 */
+	public static function get_name(): string {
+		return 'p4/page-header-img-right';
+	}
+
+	/**
+	 * Returns the pattern config.
+	 */
+	public static function get_config(): array {
+		$classname  = 'is-pattern-p4_page-header-img-right';
+		$media_link = esc_url( get_template_directory_uri() ) . '/images/placeholders/placeholder-546x415.jpg';
+		return [
+			'title'      => __( 'Page Header with image on the right', 'planet4-blocks-backend' ),
+			'categories' => [ 'page-headers' ],
+			'content'    => '
+				<!-- wp:cover {"url":"' . $media_link . '","dimRatio":0,"minHeight":550,"contentPosition":"top left","isDark":false,"className":"' . $classname . '","style":{"color":{}}} -->
+				<div class="wp-block-cover is-light has-custom-content-position is-position-top-left ' . $classname . '" style="min-height:550px">
+					<span aria-hidden="true" class="wp-block-cover__background has-background-dim-0 has-background-dim"></span>
+					<img class="wp-block-cover__image-background" alt="Default image" src="' . $media_link . '" data-object-fit="cover"/>
+
+					<div class="wp-block-cover__inner-container">
+
+					<!-- wp:group {"layout":{"type":"flex","allowOrientation":false,"justifyContent":"left","flexWrap":"wrap"}} -->
+					<div class="wp-block-group">
+						<!-- wp:heading {"level":1,"placeholder":"' . __( 'Enter title', 'planet4-blocks-backend' ) . '","style":{"typography":{"letterSpacing":"-1px"}},"textColor":"grey-80","className":"has-white-background-color has-background"} -->
```

Comment:
Same here as for the description, you probably can remove the `grey-80` text color since it's our default!